docs_outdoornav_user_manual/features/navigation.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Navigation Features
+sidebar_label: Navigation Features
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+The OutdoorNav Software contains a set of features that can be enabled
+or disabled according to your required application requirements. These
+features can be enabled or disabled through the use of environment
+variables, which should be added to the
+`/home/administrator/cpr_outdoornav_launch/outdoornav_tuning.env`. The

```

Comment:
What if we are installing on a customer computer that does not have an "administrator" account?

Reply:
Also, after changing a variable, should the customer do a power cycle for the change to take effect or is the effect immediate?